README.md

```diff
@@ -292,9 +292,9 @@ block's partial config.
 </details>
 
 <details>
-<summary>How to compose blocks? (AKA building presets)</summary>
+<summary>How to compose blocks?</summary>
 
-Got some projects with a similar, yet not identical webpack configuration? Seems like you could use a preset:
+Got some projects with similar, yet not identical webpack configurations? Create a “preset”, a function that returns a `group` of blocks so you could reuse it in several projects:
```

Comment:
Better :)

But how about "a function that returns a `group` of blocks so you *can* reuse it in *multiple* projects"?
(I will stop micro-optimizing after that, I promise ;P)